Code/Tools/ProjectManager/Source/PythonBindingsInterface.h

```diff
@@ -94,10 +94,11 @@ namespace O3DE::ProjectManager
         /**
          * Registers the gem to the specified project, or to the o3de_manifest.json if no project path is given
          * @param gemPath the path to the gem
-         * @param projectPath the path to the project. If empty, will register the external path in o3de_manifest.json 
+         * @param projectPath the path to the project. If empty, will register the external path in o3de_manifest.json
+         * @param remove Unregister instead of registering this gem 
          * @return An outcome with the success flag as well as an error message in case of a failure.
          */
-        virtual AZ::Outcome<void, AZStd::string> RegisterGem(const QString& gemPath, const QString& projectPath = {}) = 0;
+        virtual AZ::Outcome<void, AZStd::string> RegisterGem(const QString& gemPath, const QString& projectPath = {}, bool remove = false) = 0;
```

Comment:
I'm not so much a fan of using a function named "Register" to unregister even though the underlying call may be the same.  I'd rather us have a separate UnRegisterGem function. Thoughts?

Reply:
I can do that, this was just easier in the moment. Will try a solution that keeps code duplication down